internal/did/access.go

```diff
@@ -66,11 +65,19 @@ func extractKeyFromVerificationMethod(method didsdk.VerificationMethod) (pubKey
 		return
 	case method.PublicKeyJWK != nil:
 		jwkBytes, jwkErr := json.Marshal(method.PublicKeyJWK)
-		if err != nil {
+		if jwkErr != nil {
 			err = jwkErr
 			return
 		}
-		pubKey, err = jwk.ParseKey(jwkBytes)
+		parsed, parseErr := jwk.ParseKey(jwkBytes)
+		if parseErr != nil {
+			err = parseErr
+			return
+		}
+		if err = parsed.Raw(&pubKey); err != nil {
+			return
+		}
```

Comment:
Across the board, we expect `crypto.PublicKey` to be a type from go's `crypto` package. This new bit of code ensure's that's the case.